### ISSUE TITLE
Misleading var names and log messages GPIO mode

### ISSUE THREAD

**Issue by AsharLohmar** [milaq/rpi-rf#27]
In various parts of the code the lib uses the label `gpio`
```
# rpi-rf_receive
logging.info("Listening for codes on GPIO " + str(args.gpio))
# rpi_rf.py
class RFDevice:
    def __init__(self, ...):
        #...
        _LOGGER.debug("Using GPIO " + str(gpio))
```
but actually in the code of RFDevice it sets the  GPIO in mode BCM
```
GPIO.setmode(GPIO.BCM)
```
This is kind of misleading.

**Comment by palto42** [comment]
Hi @AsharLohmar
I think you mean that it's not clear which pin numbering is used:
parser.add_argument('-g', dest='gpio', type=int, default=27, help="GPIO pin (Default: 27)")
The term "pin" can be misinterpreted as BOARD/physical numbering, although it's using the GPIO/BCM numbering (too many different names and formats, there's also the wiringPi numbering).
GPIO 27 = BCM 27 = Board 13 = WiringPi 2 (on RPi1 rev.1 the Board 13 was GPIO 21)

Agree that it would be useful to update the help text to make it more clear the BCM numbering is used.
parser.add_argument('-g', dest='gpio', type=int, default=27, help="GPIO BCM number (Default: 27)")

**Comment by AsharLohmar** [comment]
Yes, that's exactly what I intended to say. Adding the BCM label here&there would be helpful.
